priv/templates/install_upgrade_escript

```diff
@@ -81,6 +82,29 @@ upgrade(DistInfo, Args) ->
 downgrade(DistInfo, Args) ->
     install(DistInfo, Args).
 
+uninstall({_RelName, NameTypeArg, NodeName, Cookie}, [VersionArg]) ->
+    TargetNode = start_distribution(NodeName, NameTypeArg, Cookie),
+    WhichReleases = which_releases(TargetNode),
+    Version = parse_version(VersionArg),
+    case proplists:get_value(Version, WhichReleases) of
+        undefined ->
+            ?INFO("Release ~s is already uninstalled.~n", [Version]);
+        old ->
+            ?INFO("Release ~s is marked old.~n", [Version]),
+            remove_release(TargetNode, Version);
+        unpacked ->
+            ?INFO("Release ~s is marked unpacked.~n", [Version]),
+            remove_release(TargetNode, Version);
+        current ->
+            ?INFO("Release ~s is marked current.~n", [Version]),
+            remove_release(TargetNode, Version);
```

Comment:
maybe we shouldn't allow the removal of a `current` version, that's a version that is already running but hasn't yet been made permanent, if the user removes it the node won't probably boot the next time